app/models/miq_ae_yaml_import.rb

```diff
@@ -222,13 +222,29 @@ def process_method(class_obj, ruby_method_file_name, method_yaml)
     if method_attributes['location'] == 'inline'
       data = load_method_ruby(ruby_method_file_name)
       method_yaml.store_path('object', 'attributes', 'data', data) if data
+    elsif method_attributes['location'] == 'playbook'
+      convert_playbook_attributes(method_attributes['options'], ruby_method_file_name)
+      method_attributes['options'].except!(:repository_name, :playbook_name, :credential_name, :vault_credential_name, :cloud_credential_name)
```

Comment:
Better to clear these name attributes inside method ```convert_playbook_attributes```.

Reply:
@lfu thanks for the review. I moved the name removing to the `convert_playbook_attributes` and added 1 more test for the situation, where is the playbook method created, then the playbook is deleted and the export is called.